frontendActions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseResetBranchToRemoteAction.java

```diff
@@ -136,6 +138,8 @@ public void actionPerformed(AnActionEvent anActionEvent) {
       }
     }
 
+    FileDocumentManager.getInstance().saveAllDocuments();
```

Comment:
Hmm `actionPerformed` is called on UI thread IIRC... is this stuff (and `doResetToRemoteWithKeep`) UI-safe? :thinking:

Reply:
(okay, the latter is just queuing a backgroundable)

Reply:
Plus add a comment in code why this has been deemed necessary in the first place

Reply:
This is fine :fire:  ("they" use `saveAllDocuments` within `actionPerformed` too)

Comment dropped